x-pack/plugins/rule_registry/server/routes/get_alert_by_id.test.ts

```diff
@@ -6,16 +6,22 @@
  */
 
 import {
+  ALERT_ID,
+  ALERT_RULE_CATEGORY,
```

Comment:
Should we import these from the `get_common_alert_fields`

Reply:
What would be the advantage? We're importing a lot of field name constants from `@kbn/rule-data-utils` anyway.